docs/guides/opera_rtc_product_guide.md

```diff
@@ -0,0 +1,247 @@
+
+OPERA's [Radiometric Terrain Corrected (RTC) Synthetic Aperture Radar (SAR) Backscatter for Sentinel-1 (S1)](https://www.jpl.nasa.gov/go/opera/products/rtc-product/ "https://www.jpl.nasa.gov/go/opera/products/rtc-product/" ){target=_blank} 
+product consists of radar backscatter normalized with respect to topography. The RTC-S1 Level-2 products are projected 
+into the appropriate UTM Zone or Polar Stereographic coordinate system for the location of each individual product, 
+and provided in Cloud-Optimized GeoTIFF (COG) format.
+
+The pixel values of the products represent gamma-0 power. The pixel spacing is 30 meters, and no speckle filter has 
+been applied.
+
+RTC products provide users with imagery of the earth's surface regardless of atmospheric conditions. This allows 
+users to monitor surface processes during natural disasters, such as hurricanes or wildfires, or in areas that are 
+prone to frequent cloud cover. Backscatter values can be used to determine surface water extent, soil moisture 
+trends, surface roughness, and vegetation conditions.
+
+#### Digital Elevation Model (DEM)
+
+Radiometric Terrain Correction requires the use of a Digital Elevation Model (DEM) both for 
+[correcting distortions caused by topography](https://storymaps.arcgis.com/stories/dc2807b444924fc3a76c117a2c909f8b#ref-n-uKhHbd "OPERA Sentinel-1 RTC StoryMap Tutorial" ){target=_blank}
+and for geocoding the output product. The OPERA RTC-S1 products are processed using the 
+[Copernicus GLO-30 DEM](https://dataspace.copernicus.eu/explore-data/data-collections/copernicus-contributing-missions/collections-description/COP-DEM "Copernicus DEM" ){target=_blank}.
+
+#### Polarization
+
+Most Sentinel-1 acquisitions are collected in two 
+[polarizations](introduction_to_sar#polarizations), 
+and the OPERA project generates RTC-S1 products for all available polarizations. Acquisitions over land generally 
+have a vertical primary polarization, meaning that the SAR signal is sent out in a vertical orientation, but both 
+the co-polarized returns (also vertically polarized) and the cross-polarized returns (horizontally polarized) are 
+measured. This results in both VV and VH RTC-S1 products. 
+
+In some areas, particularly remote islands and regions close to the Arctic Ocean, acquisitions are collected with a 
+horizontal primary polarization. This results in HH and HV RTC-S1 products. 
+
+Different polarizations are sensitive to different surface characteristics, with VV being more sensitive to surface 
+roughness, VH or HV being more sensitive to volume scatterers such as vegetation, and HH being more sensitive to 
+double-bounce scattering from objects that stand perpendicular to the ground. 
+
+### Archived OPERA RTC-S1 Products
+
+The OPERA project has generated RTC-S1 products for all Sentinel-1 acquisitions over landmasses (except Antarctica) 
+since January 1, 2022. RTC-S1 products are output for all available polarizations. As new Sentinel-1 SLCs are 
```

Comment:
Small re-wording? 
```suggestion
since January 1, 2022. RTC-S1 products are produced for all available polarizations. As new Sentinel-1 SLCs are 
```

Reply:
I was trying to avoid products being produced, and had already used generated, but I'll give the wording of this section some additional thought.

Reply:
@jacquelynsmale Does the wording change work for you? A to-do is to tighten up the expected latency interval once I have some stats.